File: desktop/src/config/providers.ts
export const PROVIDER_DISPLAY_NAMES: Record<string, string> = {
  claude: "Claude",
  codex: "Codex",
  cursor: "Cursor",
  gemini: "Gemini",
  opencode: "OpenCode",
  amp: "Amp",
};

export function getProviderDisplayName(kind: string): string {
  return PROVIDER_DISPLAY_NAMES[kind] ?? kind;
}

File: anyharness/crates/proliferate-worker/src/inventory/providers.rs
use serde_json::{json, Value};

use crate::inventory::versions::command_version;

pub fn collect() -> Option<Value> {
    Some(json!({
        "claude": command_version("claude", &["--version"]),
        "codex": command_version("codex", &["--version"]),
        "gemini": command_version("gemini", &["--version"]),
        "opencode": command_version("opencode", &["--version"])
    }))
}
